### PR TITLE
Stage

### PR DESCRIPTION
**Pull request by MathisHTW**
This frontend makes use of all the api methods which the application component offers:
translation
calculation
product and alcohol information

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=827846
```
Comment (notefox) [inline .vscode/extensions.json:2]: please add anything .vscode related to .gitignore

File: src/app/home-pic/home-pic.component.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.fitImage {
+    max-width: 99%;
```
Comment (notefox) [inline src/app/home-pic/home-pic.component.css:2]: can you explain why this is for?

**Review by notefox — CHANGES_REQUESTED** [review]
add package-lock.json and anything .vscode related to .gitignore

File: src/app/home-pic/home-pic.component.css (hunk 2 above)
Comment (MathisHTW) [reply src/app/home-pic/home-pic.component.css:2]: fitting the picture to the screen size

**Review by Ktechen — APPROVED** [review]

File: .vscode/extensions.json (hunk 1 above)
Comment (MathisHTW) [reply .vscode/extensions.json:2]: ok did refactor the gitignore

**Pull request merged**